templates/entry/edit.tpl

```diff
@@ -167,15 +167,13 @@
 
         {if $homonyms}
           <div class="form-group">
-            <label class="col-md-2">omonime</label>
+            <label for="homonyms" class="col-md-2">omonime</label>
```

Comment:
Ar trebui să existe și un element cu `id="homonyms"`, dar nu-l văd! Credeam că `<label>` este util ca să focalizeze un `<input>` la click, aici pare că nu e cazul, nu?

Reply:
Nici ` id="structStatus"` nu există, dar labelul are `for`. Dacă vrem să fim consecvenți, ar trebui revizuite toate focalizările. Unele chiar nu există și poate ar trebui.

Reply:
De acord! Nu mi-e clar cum merge focalizarea la radio buttons. Dacă face ceva rezonabil, atunci hai să-i dăm un ID. La `homonyms` cred că putem șterge _label_-ul, că nu e nimic editabil.

Reply:
L-am șters, dar am ceva probleme (din nou). Am făcut o revizie direct pe github și acum nu mă lasă să fac alte commituri la acest branch.